pom.xml

```diff
@@ -50,11 +50,17 @@
   </modules>
 
   <dependencyManagement>
-    <!-- Scalecube cluster utils -->
+
     <dependencies>
+      <!-- Scalecube cluster -->
+      <dependency>
+        <groupId>io.scalecube</groupId>
+        <artifactId>scalecube-cluster</artifactId>
+        <version>${scalecube-cluster.version}</version>
+      </dependency>
       <dependency>
-        <groupId>${project.groupId}</groupId>
-        <artifactId>scalecube-utils</artifactId>
+        <groupId>io.scalecube</groupId>
+        <artifactId>scalecube-transport</artifactId>
```

Comment:
Remove plz

Reply:
it's the main pom and these are in `dependencyManagement`

Reply:
however, yes, we can, the cluster depends on this transport